db/migrate/20160117024221_create_recommendations.rb

```diff
@@ -0,0 +1,18 @@
+class CreateRecommendations < ActiveRecord::Migration
+  def change
+    create_table :recommendations do |t|
+      t.integer :user_id, null: false
+      t.integer :recommendable_id, null: false
```

Comment:
What do you think about [foreign key constraints] here (and on other foreign keys)? Too much?

[1]: https://robots.thoughtbot.com/referential-integrity-with-foreign-keys